wagtail/migrations/0071_copy_userprofiles.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.1.8 on 2021-10-01 12:47
+
+from django.db import migrations
+
+
+def copy_userprofiles(apps, schema_editor):
+    if not apps.is_installed("wagtailusers"):
+        return
+
+    UserProfile = apps.get_model("wagtailcore.UserProfile")
+    OldUserProfile = apps.get_model("wagtailusers.UserProfile")
+    UserProfile.objects.bulk_create(
+        [
+            UserProfile(
+                user_id=user_profile.user_id,
+                submitted_notifications=user_profile.submitted_notifications,
+                approved_notifications=user_profile.approved_notifications,
+                rejected_notifications=user_profile.rejected_notifications,
+                updated_comments_notifications=user_profile.updated_comments_notifications,
+                preferred_language=user_profile.preferred_language,
+                current_time_zone=user_profile.current_time_zone,
+                avatar=user_profile.avatar,  # TODO does this copy the avatar?
+            )
+            for user_profile in OldUserProfile.objects.all()
+        ],
+        ignore_conflicts=True,
```

Comment:
`Enabling this parameter disables setting the primary key on each model instance` from the Django docs - is there a reason we need this, and don't want to try and set the pk to make migrating any FKs to `UserProfile` easier? Probably a pretty minor concern here though. (I was interested to find out that the OneToOneField on `UserProfile` isn't the primary key though! Had always assumed it would be)

Reply:
I don't, I'm afraid! Happy for it to stay as is unless we do make the `OneToOneField`-> primary key change here, in which case we should check the key is still set properly

I think that would make sense! Tbh I spotted that problem while assuming that `OneToOneField` was the primary key - that way round seems pretty natural to me

Reply:
Done!